browser-extension/mv2/src/background/background.js

```diff
@@ -1148,10 +1148,26 @@ BG.Methods.addListenerForExtensionMessages = function () {
       case RQ.EXTENSION_MESSAGES.SAVE_TEST_RULE_RESULT:
         BG.Methods.saveTestRuleResult(message, sender.tab);
         break;
+
+      case RQ.EXTENSION_MESSAGES.VIEW_RULE_IN_EDITOR:
+        BG.Methods.viewRuleInEditor(message.ruleId);
+        break;
+
+      case RQ.EXTENSION_MESSAGES.OPEN_APP_SETTINGS:
+        BG.Methods.openAppSettings();
+        break;
```

Comment:
We can directly open the URL from popup. No need to delegate it to background. You can check the `Open App` button

Reply:
@RuntimeTerror10 This has not been addressed